src/test/java/BogdanSandu/tests/Tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,82 @@
+package BogdanSandu.tests.Tema6;
+
+import BogdanSandu.driver.BrowserManager;
+import BogdanSandu.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+public class DemoQaWindow {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) {
+        navigateToDemoQAWindowPage();
+        try {
+            clickOnNewTabButton();
```

Comment:
Do you have any exceptions that can be thrown inside this try? That means to identify in one of your methods an element with a wrong id/selector/xpath. ;)